ui/serializers.py

```diff
@@ -67,8 +71,5 @@ class DropboxFileSerializer(serializers.Serializer):
     isDir = serializers.BooleanField()
 
     def create(self, validated_data):
-        video, _ = Video.objects.get_or_create(
-            s3_object_key=validated_data["name"],
-            defaults={'source_url': validated_data["link"]},
-        )
+        video, _ = Video.objects.get_or_create(defaults={'source_url': validated_data["link"]},)
```

Comment:
I still do not understand what we are trying to do here: what's the case when the object already exists?